WORKSPACE

```diff
@@ -96,9 +96,7 @@ dpkg_list(
         "python2.7-minimal",
         "libpython2.7-stdlib",
         "dash",
-        # Version required to skip a security fix to the pre-release library
-        # TODO: Remove when there is a security fix or dpkg_list finds the recent version
-        "libc-bin=2.24-11+deb9u4",
+        "libc-bin",
```

Comment:
Are we good now? Should we also update `"libc6=2.24-11+deb9u4"` above?

Reply:
Trying to figure out which packages are resolved is tricky.  Turns out we're still pulling down `2.24-1deb9u1`, not `u4`.  I'll restore libc-bin.

Reply:
The easiest way I've found to discover package versions is to look in `bazel-xxx/external/package_bundle/file/packages.bzl`

Reply:
Ah can also look at the contents of the files in `/var/lib/dpkg/status.d/` within the image.